views/standard/spec/indicators.js

```diff
@@ -214,8 +214,8 @@ define(['jquery', 'underscore', 'appearanceUtils', 'debug'], function($, _, util
          signatureStatus(pull, node, 'QA', required, pull.qa_signatures);
       },
       build_status: function status(pull, node) {
-         if (pull.status.commit_status) {
-            var commit_status = pull.status.commit_status.data;
+         if (pull.status.commit_statuses) {
+            var commit_status = pull.status.commit_statuses[0].data;
```

Comment:
We're picking an arbitrary one. It may not even be one of the required ones. Perhaps a specific one would be better, or perhaps the first one in the required list?

Oh, maybe this pull was more about dealing with the backend? Leaving the front-end changes for a future pull?

Reply:
e3e75488efa9428d121047d16b2a645ac5f8c639

Reply:
Yeah, that's another thing. There isn't an obvious answer to doing this, so it'll have to be done in a following pull.